archiver.go

```diff
@@ -28,6 +28,7 @@ const (
 var archiverAbortOnAllErrors = func(str string, fi os.FileInfo, err error) error { return err }
 var archiverAllowAllFiles = func(string, os.FileInfo) bool { return true }
 
+// Archiver is used to backup a set of directories.
```

Comment:
Where does the Archiver get that set of directories from? it's stored in the `*server.Server` somehow? Maybe you can say "Archiver is used to back up a set of directories as references by a Server" or something like that?